### PR TITLE
added deployment documentations and faq

### PR DESCRIPTION
**Pull request by natmasslove**

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -1,61 +1,83 @@
...
+*`stage-name` is a parameter which identifies the deployment stage (usually dev/test/qa/uat/prod, but can be any string identifier of your choice).*
+
+What happens during *`cdk deploy`* of tooling environment:
+- Configuration files are validated (e.g. if delivery methods referred to in *recipients.json* are aligned with those defined in *general.json*)
+- Configuration files are copied to S3 bucket (where all SALMON components will read the configuration from).
+- AWS resources are created. For a list of resources, please refer to [Architecture document](/docs/architecture.md).
+
+*Note: If you change the settings (e.g. add a new recipient), you will need to run the aforementioned "cdk deploy" command to validate new config files and upload them onto S3.*
+
+*Note: If you choose to use optional Grafana component, make sure you to have a VPC with a public subnet and a security group prepared. The security group should allow access to port 3000 from any IP address.*
```
Comment (aryna-yukhnevich) [inline docs/deployment.md:71]: would it be useful to mention how to deploy-destroy only Grafana stack ?
cdk deploy cf-salmon-InfraToolingGrafanaStack-<your_stage_name>  --context stage-name=<your_stage_name>
cdk destroy cf-salmon-InfraToolingGrafanaStack-<your_stage_name> --context stage-name=<your_stage_name>

Comment (aryna-yukhnevich) [reply docs/deployment.md:71]: or probably we can mention this case in FAQ

**Review by aryna-yukhnevich — APPROVED** [review]
looks good to me, thanks!

File: docs/deployment.md (hunk 1 above)
Comment (natmasslove) [reply docs/deployment.md:71]: That's a good point. I'm planning to get rid of this nuance in ver 1.1 (using Nested Stacks, probably).
There will be a follow-up task

**Pull request merged**